### PR TITLE
Remove unused, ZooKeeper-related things

### PR DESCRIPTION
**Pull request by izrik**
This PR removes code and configuration items related to ZooKeeper. I gather that the original plan was to use ZK to coordinate some work on shards; however, it looks like it was never used. There aren't any zookeeper instances in staging or prod, that I could find.

The two tests that supposed to test it were explicitly disabled in the pom file, so they never run. I took them out as well.

After removing the unused code, this puts our code coverage at about 73%.

### CODE REVIEW

**Comment by izrik** [comment]
@drenalin23 @shintasmith @ChandraAddala @VinnyQ @usnavi Please review and comment.

File: blueflood-udp/src/main/java/com/rackspacecloud/blueflood/service/udp/MainIngestor.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ private static void checkConfiguration() {
 
         log.info("Configuration is good");
 
-        boolean useZookeeper = !"NONE".equals(config.getStringProperty(CoreConfig.ZOOKEEPER_CLUSTER));
+        boolean useZookeeper = false;
         log.info("Using zookeeper? " + useZookeeper);
```
Comment (usnavi) [inline blueflood-udp/src/main/java/com/rackspacecloud/blueflood/service/udp/MainIngestor.java]: can we delete the boolean and this log statement?  I think that's the only use of the variable.

**Comment by usnavi** [comment]
After this is accepted we'll need to create a story to remove any ZK-related stuff out of blueflood-chef.  Looks like there are some minor references with rollup_redundancy which we can remove.

File: blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java [hunk 2]
```diff
@@ -1,76 +0,0 @@
-/*
```
Comment (shintasmith) [inline blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java]: Why is the ScheduleContextIntegrationTest removed?

Comment (shintasmith) [inline blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java]: I think ScheduledContext is a real class that we still use. Maybe we shouldn't be removing the whole file, but only the parts that uses ZooKeeper?

Comment (shintasmith) [inline blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java]: Oh, I see that the 2 tests in ScheduledContext are related to ZooKeeper only. There are no other tests for ScheduledContext. 😱 I am fine adding the real test for ScheduledContext in a different story, if needed.

**Comment by shintasmith** [comment]
After you address @usnavi 's comment, I'm good with you merging this. +1

File: blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java (hunk 2 above)
Comment (izrik) [inline blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/service/ScheduleContextIntegrationTest.java]: There is also a unit test called `ScheduledContextTest`. This integration test only tests the stuff related to ZK.

File: blueflood-udp/src/main/java/com/rackspacecloud/blueflood/service/udp/MainIngestor.java (hunk 1 above)
Comment (izrik) [inline blueflood-udp/src/main/java/com/rackspacecloud/blueflood/service/udp/MainIngestor.java]: Can do.

Comment (izrik) [inline blueflood-udp/src/main/java/com/rackspacecloud/blueflood/service/udp/MainIngestor.java]: @usnavi Done.

**Comment by usnavi** [comment]
lgtm

**Comment by usnavi** [comment]
@izrik - the issue with travis is fixed.  Please rebase to add the fix to your branch and cause travis to rerun the build.

**Comment by izrik** [comment]
Done.

**Pull request merged**